docs/en/contribution/How-to-release.md

```diff
@@ -10,7 +10,7 @@ This documentation guides the release manager to release the SkyWalking Python i
 
 ## Add your GPG public key to Apache svn
 
-1. Upload your GPG public key to a public GPG site, such as [MIT's site](http://pgp.mit.edu:11371/). 
+1. Upload your GPG public key to a public GPG site, such as [Ubuntu's GPG keyserver](https://keyserver.ubuntu.com/). 
```

Comment:
Is this an official rename?

Reply:
Or simply mit one is gone? 
The reason I asked is about Maven release, the nexus will use some gpg servers to access public keys for verification. This is why it is mentioned in Java release, and copied here.

Reply:
> Or simply mit one is gone? The reason I asked is about Maven release, the nexus will use some gpg servers to access public keys for verification. This is why it is mentioned in Java release, and copied here.

I think the MIT one is gone and won't come back any more. Last time I did Python agent release, the MIT site was already broken so I used the Ubuntu keyserver and it worked perfectly.

Reply:
Uploading gpg key to a public server is not required in Python agent release process, let's simply remove this in the doc @Superskyyy

Reply:
Sure!

Reply:
> Uploading gpg key to a public server

The other step "upload the key to Apache svn " is still needed 🤭